documents-d-archeologie-francaise.csl

```diff
@@ -424,214 +406,164 @@
   </citation>
   <bibliography>
     <sort>
-      <key macro="responsability"/>
+      <key macro="author-short"/>
       <key macro="year-date"/>
       <key macro="title"/>
     </sort>
     <layout>
-      <group prefix="" suffix="" delimiter=" ">
+      <group prefix="" suffix="&#9;" delimiter=" ">
```

Comment:
In fact, it is a "normal" behavior in Word: a tabulation is needed before a line feed if you don't want the whole line to be justified:
![word](https://cloud.githubusercontent.com/assets/2558668/12447296/770b3cb2-bf6f-11e5-828d-9bab47fedc2a.PNG)
On the other hand, OpenOffice doesn't show this behavior:
![tab openoffice](https://cloud.githubusercontent.com/assets/2558668/12447309/9048dc8e-bf6f-11e5-9fd6-e681c2ad8015.PNG)
In the peculiar case of this style, `display="block` should not be applied to the first (author-date) group.